src/content_sao.cpp

```diff
@@ -1524,3 +1558,22 @@ float PlayerSAO::getZoomFOV() const
 {
 	return m_prop.zoom_fov;
 }
+
+void PlayerSAO::setPhysicsModifier(
+	const std::string& key, const PhysicsModifier& modifier)
+{
+	dirtyPhysicsModifier();
+	m_physics_modifiers[key] = modifier;
+}
+
+void PlayerSAO::deletePhysicsModifier(
+	const std::string& key)
```

Comment:
Misplaced reference, maybe also move this line up (within the 80 character limit for sure)